mail_tracking/tests/test_gc_mail_tracking_email.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields
+from odoo.tests.common import SavepointCase
+
+
+class TestMailTrackingEmailCleanUp(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.settings = cls.env["res.config.settings"].create(
+            {"enable_old_mail_tracking_email_deletion": True}
+        )
+        cls.settings.execute()
```

Comment:
Always use `set_values` instead of `execute` as it does too many things that will slow down the setup

Reply:
Changed to `set_values`